waiter/resources/web/oauth.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<html>
+<head>
+    <title>Login - Waiter</title>
+    <meta http-equiv="x-ua-compatible" content="IE=edge" />
+    <style type="text/css">
+   html {
+     font-family: 'Lato', 'Helvetica Neue', Arial, Helvetica, sans-serif;
+     color: rgba(0,0,0,.87);
+   }
+
+   .main-content {
+     margin: auto;
+     margin-top: 4em;
+     width: 800px;
+   }
+
+   .main-note {
+     color: #0e566c;
+     text-align: center;
+   }
+
+    </style>
+</head>
+<body>
+
+<div class="main-content">
+    <h2 class="main-note">
+        Login to Waiter
+    </h2>
+    <div>
+        <p>Choose a provider:</p>
+        <% (doseq [[provider-key provider] providers] %>
+            <li><a class="redirect" href="/waiter-auth/oauth/<%= (name provider-key) %>/redirect/<%= path %><% (when query-string %><%= (str \? query-string) %><% ) %>"><%= (display-name provider) %></a></li>
```

Comment:
Query: why does `(display-name provider)` not need the namespace qualification?

Reply:
In `oauth.clj`
 `:display-name (fn [provider] (display-name provider))`